rotkehlchen/chain/manager.py

```diff
@@ -1248,3 +1248,14 @@ def get_eth2_history_events(
                 ))
 
         return defi_events
+
+    @protect_with_lock()
```

Comment:
the lock is not needed here but in the loopring module itself, since it's simultaneous access to that function that you want to guard against. The other place it's called is from the api here:

https://github.com/rotki/rotki/blob/ff9138a796287e8eac7dfb3fa2b03700c657fb51/rotkehlchen/api/rest.py#L2273-L2281